src/ui/layout/mct-tree.vue

```diff
@@ -189,21 +187,30 @@ export default {
             return {
                 paddingLeft: offset + 'px'
             };
+        },
+        ancestorsHeight() {
+            if (this.activeSearch) {
+                return 0;
+            }
+
+            return this.itemHeight * this.ancestors.length;
+        },
+        pageThreshold() {
+            return Math.ceil(this.availableContainerHeight / this.itemHeight) + ITEM_BUFFER;
+        },
+        scrollActive() {
+            return !(this.childrenHeight > this.availableContainerHeight);
         }
     },
     watch: {
         syncTreeNavigation() {
-            if (this.isLoading) {
-                return;
-            }
-
             const AND_SAVE_PATH = true;
             let currentLocationPath = this.openmct.router.currentLocation.path;
             let hasParent = this.currentlyViewedObjectParentPath() || (this.multipleRootChildren && !this.currentlyViewedObjectParentPath());
             let jumpAndScroll = currentLocationPath
                     && hasParent
                     && !this.currentPathIsActivePath();
-            let justScroll = this.currentPathIsActivePath() && !this.noScroll;
+            let justScroll = this.currentPathIsActivePath() && this.scrollActive;
```

Comment:
Booleans should always start with `is`, `are`, etc.

Reply:
Got rid of this guy, but I will make note of that.